### ISSUE TITLE
Review color contrast for a11y

### ISSUE THREAD

**Issue by Kriesse** [cssconf/2018.cssconf.eu#46]
a few color combinations on the website fail a11y checks for colorblindness and low vision.

- [x] provide an updated design
- [ ] implement it

**Comment by RalloTubbs** [comment]
links: change normal state with hover, so normal links are red

Footer: screenshot shows the new colours
![screenshot 2017-11-30 19 37 00](https://user-images.githubusercontent.com/7797904/33448376-29c9e80e-d606-11e7-9cef-a980417a6434.png)

**Comment by Kriesse** [comment]
@RalloTubbs done!

The remaining issue is the button, which colors do you suggest do make it easier to read for color-blind people? (I have actually trouble with the hover state myself):

## normal
![image](https://user-images.githubusercontent.com/53684/33477588-784c0446-d686-11e7-8aa3-7dba88fe1ea3.png)

## hover
![image](https://user-images.githubusercontent.com/53684/33477595-7e6fd41a-d686-11e7-914b-dfa5c551c510.png)

**Comment by RalloTubbs** [comment]
who made this hover :)
I will have look latest tomorrow, since then you could remove the hover.

**Comment by Kriesse** [comment]
👍

**Comment by Kriesse** [comment]
@RalloTubbs 
A preliminary solution for the button is live: https://2018.cssconf.eu/
Should we leave it like this, or do you have another color suggestion? If so, let me know, it's a quick fix!

**Comment by Kriesse** [comment]
Closing, contrast is fixed. Let's open a new design issue if the colors don't work for @RalloTubbs
